source/chapters/library.rst

```diff
@@ -894,6 +899,39 @@ different location. Select :guilabel:`Use Default Library` to reset.
   Mac partition mounted in Linux, you can load it and use your iTunes tracks and
   playlists as well.
 
+.. _library-rekordbox:
+
+Using the Rekordbox library
+---------------------------
+To read Rekordbox databases for CDJ/XDJ players on USB or SD devices,
+the database has to be exported to a USB or SD device with a FAT or HFS
+file system using the Rekordbox Export mode. Instructions on how to do this can be found in this `article by Pioneer DJ <https://forums.pioneerdj.com/hc/en-us/articles/205021549-Moving-your-rekordbox-database-Video->`__ .
```

Comment:
I don't think this is the right article. It's about moving your DB rather and USB export. However, the Pioneer documentation is pretty bad and I can t find a better link. Maybe we should just remove the last sentence.

Reply:
I failed to find proper Pioneer DJ documentation too. Let me remove  it.